client/src/util/storage.ts

```diff
@@ -67,7 +67,17 @@ export function saveCssPreference(css: UserTheme) {
 }
 
 export function loadCssPreference(): UserTheme {
-  return loadFromStorageAsString(window.localStorage, "cssTheme", "lightTheme") as UserTheme;
```

Comment:
The culprit was the default value for UserTheme, which was "lightTheme" instead of "light".

I rewrote this a bit, so Typescript will make sure through the type system that only "light" or "dark" can be returned (also covers cases where some different string is saved in localStorage).